src/stcal/resample/utils.py

```diff
@@ -143,6 +143,24 @@ def build_driz_weight(model, weight_type=None, good_bits=None,
     elif weight_type == 'exptime':
         exptime, s = get_tmeasure(model)
         result = exptime * dqmask
+        
+    elif weight_type == "ivsky":
+        var_sky = model["var_sky"]
+        if (
+            var_sky is not None and 
+            var_sky.shape == data.shape
+        ):
+            with np.errstate(divide="ignore", invalid="ignore"):
+                inv_sky_variance = var_sky**-1
+            inv_sky_variance[~np.isfinite(inv_sky_variance)] = 0
+        else:
+            warnings.warn(
+                "'var_sky' array is not available. "
+                "Setting drizzle weight map to 1",
+                RuntimeError,
+            )
```

Comment:
```suggestion
            warnings.warn(
                "'var_sky' array is not available. "
                "Setting drizzle weight map to 1",
                RuntimeError,
            )
```
I think it would be more consistent with the current code if this was not a warning. Instead `validate_input_model` could require `var_sky` if `weight_type==ivsky` as is the case for `ivm` and `var_rnoise`:
https://github.com/spacetelescope/stcal/blob/e34115bf6d06bf5f636dd5ebe8a2b12974f0b023/src/stcal/resample/resample.py#L687-L689

Reply:
Done.